macosx/build-dmg.sh

```diff
@@ -77,7 +77,7 @@ echo
 VERSION=$(${PYTHON} -c 'import os
 os.chdir("..")
 from frescobaldi_app import appinfo
-print(appinfo.version)')
+print(appinfo.running_version())')
```

Comment:
Please don't change this: here I am retrieving the version of Frescobaldi that will appear in the compiled application for the release, but I will always build the release from the Git repository, so `appinfo.running_version()` will return the version with the `-git` suffix, which I do not want.